app/lib/publications.js

```diff
@@ -0,0 +1,396 @@
+    publications: [
+      {citation: "Cyclone Center: Can Citizen Scientists Improve Tropical Cyclone Intensity Records?, Hennon+ 2014.",
+      href: "http://journals.ametsoc.org/doi/abs/10.1175/BAMS-D-13-00152.1",
+      date: "June 1, 2015"}
+    ]}
+  ],
+  humanities: [
+    {slug: "zooniverse/ancient-lives",
+    publications: [
+      {citation: "A computational pipeline for crowdsourced transcriptions of Ancient Greek papyrus fragments, Williams+ 2014.",
+      href: "http://ieeexplore.ieee.org/xpl/articleDetails.jsp?reload=true&tp=&arnumber=7004460&queryText%3Dwilliam+alex+pridemore",
+      date: "Oct 27, 2014"},
+      {citation: "P.Oxy 5156, Plutarch Moralia 660C, 661B-C (Quaestiones Convivales IV PR., 1.2), in The Oxyrhynchus Papyri Vol. 78, Brusuelas 2013.",
+      date: "2013"}
+    ]},
+    {slug: "zooniverse/operation-war-diary",
+    publications: [
+      {citation: "A Life in the Trenches? The Use of Operation War Diary and Crowdsourcing Methods to Provide an Understanding of the British Army’s Day-to-Day Life on the Western Front, Grayson+ 2016.",
+      href: "http://bjmh.org.uk/index.php/bjmh/article/view/96",
+      date: "2016"}
+    ]}
+  ],
+  nature: [
+    {slug: "zooniverse/whale-fm",
+    publications: [
+      {citation: "Classification of large acoustic datasets using machine learning and crowdsourcing: Application to whale calls, Shamir+ 2014.",
+      href: "http://scitation.aip.org/content/asa/journal/jasa/135/2/10.1121/1.4861348",
+      date: "Dec 17, 2013"},
+      {citation: "Repeated call types in short-finned pilot whales, Globicephala macrorhynchus, Sayigh+ 2012.",
+      href: "http://onlinelibrary.wiley.com/doi/10.1111/j.1748-7692.2012.00577.x/abstract",
+      date: "July 9, 2012"}]
+    },
+    {slug: "zooniverse/snapshot-serengeti",
+    publications: [
+      {citation: "Applying a random encounter model to estimate lion density from camera traps in Serengeti National Park, Tanzania, Cusack+ 2015.",
+      href: "http://onlinelibrary.wiley.com/doi/10.1002/jwmg.902/full",
+      date: "May 28, 2015"},
+      {citation: "Snapshot Serengeti, high-frequency annotated camera trap images of 40 mammalian species in an African savanna, Swanson+ 2015.",
+      href: "http://www.nature.com/articles/sdata201526",
+      date: "June 9, 2015"}]
+    }
+  ],
+  medicine: [
+    {slug: "zooniverse/cell-slider",
+    publications: [
+      {citation: "Crowdsourcing the General Public for Large Scale Molecular Pathology Studies in Cancer, Candido dos Reis+ 2015.",
+      href: "http://www.ebiomedicine.com/article/S2352-3964(15)30016-5/pdf",
+      date: "May 9, 2015"}]
+    },
+  ],
+  meta: [
+    {name: "Meta Studies",
+    publications: [
+      {citation: "Intervention Strategies for Increasing Engagement in Crowdsourcing: Platform, Predictions, and Experiments, Segal+ 2016."
+      href: "http://research.microsoft.com/en-us/um/people/horvitz/engagement_intervention.pdf"
```

Comment:
The entry in `Publications.meta.publications` titled "Intervention Strategies for Increasing Engagement in Crowdsourcing: Platform, Predictions, and Experiments, Segal+ 2016." is missing end-of-line commas for `citation` and `href`, making the Publications object invalid.

This resulted - when testing on localhost with OSX+Chrome53 - in a crash (no app loading) and the console error, `Uncaught Error: Cannot find module "../../lib/publications" on news-pullout.jsx`. (Staging works fine though, so I'm guessing the latest branch commit isn't what's staged.)